program/rust/src/accounts/price.rs

```diff
@@ -111,17 +122,15 @@ mod price_pythnet {
             }
         }
         /// This function gets triggered when there's a succesful aggregation and updates the cumulative sums
-        pub fn update_price_cumulative(&mut self) -> Result<(), OracleError> {
+        pub fn update_price_cumulative(&mut self) {
+            debug_assert!(self.agg_.status_ == PC_STATUS_TRADING);
```

Comment:
hmmm why would we need the debug assert here :?

Reply:
Our assumption is that this check should always succeed when we call `update_price_cumulative()`, no?

Reply:
Removed debug assert.